tutu-frontend/src/lib/types/perustelu.ts

```diff
@@ -1,8 +1,8 @@
 export type Perustelu = {
   id: string;
   hakemusId: string;
-  virallinenTutkinnonMyontaja?: boolean;
-  virallinenTutkinto?: boolean;
+  virallinenTutkinnonMyontaja?: boolean | undefined;
+  virallinenTutkinto?: boolean | undefined;
```

Comment:
```suggestion
  virallinenTutkinnonMyontaja?: boolean;
  virallinenTutkinto?: boolean;
```
undefined ei tartte mun mielestä olla kun tuo kysymysmerkki kertoo sen jo.